contracts/Edition.sol

```diff
@@ -33,18 +33,20 @@ contract Edition is
     OwnableUpgradeable
 {
     struct EditionState {
-        // Total size of edition that can be minted
-        // uint64 is still billions of billions, should be enough for everyone
-        uint56 editionSize;
-        // the edition can be minted up to this timestamp in seconds -- 0 means no end date
-        // uint56 is enough for billions of years
-        uint56 endOfMintPeriod;
-        // Royalty amount in bps (uint16 is large enough to store 10000 bps)
-        uint16 royaltyBPS;
         // how many tokens have been minted (can not be more than editionSize)
         uint56 numberMinted;
         // how many tokens have been burned (can not be more than numberMinted)
         uint56 numberBurned;
+        // Pad to make the state fit in exactly 256 bits
+        uint16 __pad;
```

Comment:
so it's cheaper to add blank bytes in the struct ? 🤯

Reply:
> we use it to store 5 values that were previously 256 bits each

Indeed ! Didn't notice those integers were previously 256 bits !

Reply:
yeah there will be blank bytes anyway because each storage slot is 32 bytes. Adding the padding explicitly lets me control where the blank bytes go